doc/vundle.txt

```diff
@@ -263,6 +271,12 @@ Installs or updates the configured plugins. Press 'u' after updates complete
 to see the change log of all updated bundles. Press 'l' (lowercase 'L') to
 see the log of commands if any errors occurred.
 
+To update specific plugins, write their names separated by space:
+>
+  :PluginInstall! vim-surround vim-fugitive
```

Comment:
Above, the full tpope/vim-surround is used, but here only vim-surround. Why the distinction?